### PR TITLE
Add tests for cmd module

### PR DESCRIPTION
**Pull request by larsks**
- Add some basic tests for subcommands. These primarily test responses
  to valid and invalid options.
- Add coverage targets to Makefile

Depends on #10
Depends on #11 
Depends on #12

### CODE REVIEW

**Comment by HumairAK** [comment]
dependencies were merged, rebase please : D

**Comment by larsks** [comment]
@HumairAK rebased!

File: cmd/enableMonitoring_test.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+		constants.ComponentPath,
+		"monitoring-rbac",
+	), 0755)
+	assert.Nil(err)
+
+	// Should fail due to missing kustomization.yaml
+	cmd = NewCmdEnableMonitoring(suite.api)
+	cmd.SetArgs([]string{"arg1"})
+	err = cmd.Execute()
+	assert.Contains(err.Error(), "kustomization.yaml: no such file or directory")
```
Comment (HumairAK) [inline cmd/enableMonitoring_test.go:67]: shouldn't you check for `err != nil` before accessing a member `err.Error()`?

Comment (larsks) [reply cmd/enableMonitoring_test.go:67]: Yeah, probably. I'll fix that.

**Review by HumairAK — APPROVED** [review]
/lgtm
/approve

**Pull request merged**